apps/aecore/lib/aecore/oracle/tx/oracle_query_tx.ex

```diff
@@ -263,17 +261,13 @@ defmodule Aecore.Oracle.Tx.OracleQueryTx do
     tx_fee_is_met =
       case query_ttl do
         %{ttl: ttl, type: :relative} ->
-          fee >= calculate_minimum_fee(ttl)
+          fee - GovernanceConstants.oracle_query_base_fee() >= Oracle.calculate_minimum_fee(ttl)
 
         %{ttl: _ttl, type: :absolute} ->
-          if block_height != nil do
-            fee >=
-              query_ttl
-              |> Oracle.calculate_ttl(block_height)
-              |> calculate_minimum_fee()
-          else
-            true
-          end
+          fee - GovernanceConstants.oracle_query_base_fee() >=
```

Comment:
maybe split the minus into a named variable before the greater equals check, eventually put the greater equals check in a variable as well to start the piping from

Reply:
same applies to other parts of the code